# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/transportbridge/MediaPipeline.h
# Commit: d838dc7d2bb8
# Full Hash: d838dc7d2bb89ab5c9c2e8e93895db1c9310560f
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Update include paths for moved upstream code. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130059
# ==============================================================================

diff -r 587c40771588 -r d838dc7d2bb8 dom/media/webrtc/transportbridge/MediaPipeline.h
--- a/dom/media/webrtc/transportbridge/MediaPipeline.h	Fri Oct 23 13:49:51 2020 -0400
+++ b/dom/media/webrtc/transportbridge/MediaPipeline.h	Fri Oct 23 14:02:54 2020 -0400
@@ -24,7 +24,7 @@
 #include "MediaSegment.h"
 #include "jsapi/PacketDumper.h"
 
-#include "webrtc/modules/rtp_rtcp/include/rtp_header_parser.h"
+#include "test/rtp_header_parser.h"
 
 // Should come from MediaEngine.h, but that's a pain to include here
 // because of the MOZILLA_EXTERNAL_LINKAGE stuff.